packages/vue3/tests/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
+  "compilerOptions": {
```

Comment:
why do we need two tsconfigs?

Reply:
Yes, the tests one is for having types while working on tests.
It's a simple solution for not including tests in the main tsconfig, and so not generating unneeded d.ts for non src files when building.